CMakeLists.txt

```
@@ -130,6 +131,13 @@ if(Torch_DOWNLOADED)
       install(DIRECTORY ${Torch_LINK_DIR} DESTINATION . COMPONENT libneml2)
 endif()
 
+# ----------------------------------------------------------------------------
+# Intel Extension for PyTorch
+# ----------------------------------------------------------------------------
+if (AURORA_BUILD)
+      find_library(IPEX_LIB intel-ext-pt-gpu PATHS ${INTEL_EXTENSION_FOR_PYTORCH_PATH}/lib NO_DEFAULT_PATH REQUIRED)
```

```suggestion
      find_library(IPEX_LIB intel-ext-pt-gpu)
```
I suspect something like this will just work. You can then specify the path using `-DIPEX_LIB_ROOT=...`.